tests/test_measure.py

```diff
@@ -138,32 +143,33 @@ def testFootprintsMeasure(self):
         schema = afwTable.SourceTable.makeMinimalSchema()
         task = measBase.SingleFrameMeasurementTask(schema, config=measureSourcesConfig)
         measCat = afwTable.SourceCatalog(schema)
+        footprints.makeSources(measCat)
         # now run the SFM task with the test plugin
         sigma = 1e-10
         psf = algorithms.DoubleGaussianPsf(11, 11, sigma)  # i.e. a single pixel
         self.exposure.setPsf(psf)
         task.run(measCat, self.exposure)
 
+        self.assertGreater(len(measCat), 0)
```

Comment:
> Did you decide against checking against len(flux)?

Nah; I just forgot, because I haven't had my coffee yet this morning.

> Can/should a check like this be added to the testDetection function?

I'd not looked. Doesn't seem like it could do any harm, so I've pushed one. I can't check locally because I don't have an afwdata, so I'll wait for Jenkins (#29298).